BossMod/Components/Chains.cs

```diff
@@ -34,6 +45,7 @@ public override void OnTethered(Actor source, ActorTetherInfo tether)
             {
                 SetPartner(source.InstanceID, target);
                 SetPartner(target.InstanceID, source);
+                _initialDistance = (source.Position - target.Position).Length();
```

Comment:
this should probably be per-partner, like the `_partner` array is. you could even make that field a `(Actor? Partner, float InitialDistance)[]` instead. otherwise, if 4 people get chained at once, the initial distance will be effectively random, it'll just get set to whoever received the chain event last